editor/themes/editor_theme_manager.cpp

```diff
@@ -2246,6 +2185,80 @@ void EditorThemeManager::_populate_editor_styles(const Ref<EditorTheme> &p_theme
 		p_theme->set_type_variation("ItemListSecondary", "ItemList");
 	}
 
+	// Editor GUI widgets.
+	{
+		// EditorSpinSlider.
+		p_theme->set_color("label_color", "EditorSpinSlider", p_config.font_color);
+		p_theme->set_color("read_only_label_color", "EditorSpinSlider", p_config.font_readonly_color);
+		p_theme->set_color(SceneStringName(font_color), "EditorSpinSlider", p_config.font_color);
+		p_theme->set_color("font_unedited_color", "EditorSpinSlider", p_config.font_readonly_color);
+
+		p_theme->set_font(SceneStringName(font), "EditorSpinSlider", p_theme->get_font(SceneStringName(font), SNAME("LineEdit")));
+		p_theme->set_font_size(SceneStringName(font_size), "EditorSpinSlider", p_theme->get_font_size(SceneStringName(font_size), SNAME("LineEdit")));
+
+		p_theme->set_constant("updown_v_separation", "EditorSpinSlider", 0);
+
+		p_theme->set_stylebox(CoreStringName(normal), "EditorSpinSlider", p_theme->get_stylebox(CoreStringName(normal), "LineEdit")->duplicate());
+		p_theme->set_stylebox("read_only", "EditorSpinSlider", p_theme->get_stylebox("read_only", "LineEdit")->duplicate());
+		p_theme->set_stylebox("focus", "EditorSpinSlider", p_config.button_style_focus);
+		Ref<StyleBoxFlat> editor_spin_label_bg = p_config.base_style->duplicate();
+		editor_spin_label_bg->set_bg_color(p_config.dark_color_3);
+		editor_spin_label_bg->set_border_width_all(0);
+		p_theme->set_stylebox("label_bg", "EditorSpinSlider", editor_spin_label_bg);
+		p_theme->set_stylebox("updown_hovered", "EditorSpinSlider", p_theme->get_stylebox(SceneStringName(hover), "FlatMenuButton"));
```

Comment:
Moved `Editor GUI widgets` down to be able to use `FlatMenuButton` styles, but forgot to duplicate them.